cpp/include/cudf/column/column.hpp

```diff
@@ -159,6 +159,13 @@ class column {
    */
   [[nodiscard]] size_type size() const noexcept { return _size; }
 
+  /**
+   * @brief Returns the number of data bytes
```

Comment:
This should be clearer. Is it the number of bytes of a single element? `size() * sizeof(T)`?

Reply:
It would be `size() * sizeof(T)` but only where `T` is a fixed-width-type.
The intention here is to expose the actual device-buffer `size()` which may be larger than `sizeof(size_type)` in the solutions we are looking at ways to allow the total number of bytes in a strings column to exceed `size_type`.